lib/testers/rule-tester.js

```diff
@@ -429,7 +429,7 @@ RuleTester.prototype = {
                     item.errors.length, item.errors.length === 1 ? "" : "s", messages.length, util.inspect(messages)));
 
                 for (let i = 0, l = item.errors.length; i < l; i++) {
-                    assert.ok(!("fatal" in messages[i]), `A fatal parsing error occurred: ${messages[i].message}`);
+                    assert.ok(!messages[i].fatal, `A fatal parsing error occurred: ${messages[i].message}`);
```

Comment:
This makes it look like to me like it's a design choice not to include if if not actually fatal. Why?